Cargo.toml

```diff
@@ -14,40 +14,40 @@ license = "MIT"
 readme = "./README.md"
 repository = "https://github.com/mario-eth/soldeer"
 rust-version = "1.80"
-version = "0.4.1"
+version = "0.5.0"
 
 [workspace.lints.clippy]
 dbg-macro = "warn"
 manual-string-new = "warn"
-uninlined-format-args = "warn"
-use-self = "warn"
 redundant-clone = "warn"
+uninlined-format-args = "warn"
 unwrap_used = "warn"
+use-self = "warn"
 
 [workspace.lints.rust]
+redundant-lifetimes = "warn"
 rust-2018-idioms = "warn"
 unreachable-pub = "warn"
 unused-must-use = "warn"
-redundant-lifetimes = "warn"
 
 [workspace.lints.rustdoc]
 all = "warn"
 
 [workspace.dependencies]
 bon = "2.0.0"
-clap = { version = "4.5.9", features = ["derive"] }
+clap = {version = "4.5.9", features = ["derive"]}
 cliclack = "0.3.4"
-derive_more = { version = "1.0.0", features = ["from", "display", "from_str"] }
+derive_more = {version = "1.0.0", features = ["from", "display", "from_str"]}
 mockito = "1.5.0"
 path-slash = "0.2.1"
 rayon = "1.10.0"
-reqwest = { version = "0.12.5", default-features = false }
-temp-env = { version = "0.3.6", features = ["async_closure"] }
+reqwest = {version = "0.12.5", default-features = false}
+temp-env = {version = "0.3.6", features = ["async_closure"]}
 testdir = "0.9.1"
 thiserror = "1.0.63"
-tokio = { version = "1.38.0", features = [
-    "io-util",
-    "macros",
-    "process",
-    "rt-multi-thread",
-] }
+tokio = {version = "1.38.0", features = [
+  "io-util",
+  "macros",
+  "process",
+  "rt-multi-thread",
+]}
```

Comment:
which formatter are you using for TOML? I think cargo puts spaces around the curly braces which is probably good to keep. Also the indenting level is different.